docs/conceptual-guides/shadowing.md

```diff
@@ -0,0 +1,72 @@
+---
+myst:
+  html_meta:
+    "description": "How to shadow a component or module in Seven"
+    "property=og:description": "How to shadow a component or module in Seven"
+    "property=og:title": "Shadow a component or module"
+    "keywords": "Seven, shadow, override, component, frontend"
+---
+
+# Shadow a component or module in Seven
+
+Component or module shadowing allows you to override a component from an existing package in a clean and structured way, without directly modifying third-party code. This technique is particularly useful for customizing the behavior or appearance of specific components in Seven or any add-ons.
+
+This mechanism is powered by Vite's `resolve.alias`, which maps a module path to an alternative file or folder. The override is resolved **at build time**, replacing the original module with your custom implementation.
+
+## How it works
+
+1. Locate the component you want to override (a.k.a. "shadow").
+2. Replicate the original folder structure within a `customizations` folder inside your project.
+3. Vite will resolve the alias to your version during the build.
+
+## Project structure for shadowing
+
+Suppose the component you want to shadow is located at:
+
+```
+node_modules/@plone/slots/components/Logo/Logo.svg
+```
+
+You will need to replicate this structure under `src/customizations`, like so:
+
+```
+customizations/
+└── @plone/
+    └── slots/
+        └── components/
+            └── Logo/
+                └── Logo.svg
+```
+
+> Tip: Use absolute imports in your shadowed modules to avoid resolution issues.
```

Comment:
````suggestion
```{tip}
Use absolute imports in your shadowed modules to avoid resolution issues.
```
````